lib/lita/adapters/slack/api.rb

```diff
@@ -81,17 +93,21 @@ def set_topic(channel, topic)
           call_api("channels.setTopic", channel: channel, topic: topic)
         end
 
-        def rtm_start
-          response_data = call_api("rtm.start")
+        def rtm_connect
+          response_data = call_api("rtm.connect")
+          channels = conversations_list('public_channel,private_channel,im')
+
+          users = SlackUser.from_data_array(users_list)
+          user = users.find { |user| user.id == response_data['self']['id'] }
```

Comment:
We now need to make a separate request to get the `users.list`.

Also, `rtm.connect` does not contain the same `user` data as `rtm.start`. So, I opted to find the user in the already fetched user list.